packages/dd-trace/src/appsec/iast/taint-tracking/rewriter-telemetry.js

```diff
@@ -4,39 +4,10 @@ const iastTelemetry = require('../telemetry')
 const { Verbosity } = require('../telemetry/verbosity')
 const { INSTRUMENTED_PROPAGATION } = require('../telemetry/iast-metric')
 
-const telemetryRewriter = {
```

Comment:
Is there a reason you deleted the whole telemetry part ?

Reply:
This _should_ be restored elsewhere. I'll double-check that.

Reply:
Yeah, so this entire file was just to turn telemetry on and off based on verbosity. I found it very confusing and redundant, so I replaced most of it with just line 97 in `rewriter.js`. I _think_ that preserves functionality. The tests needed to be removed because they were just for the `on/off` mechanism.

I needed to ensure appropriate args were being passed along and this wrapper was making it all confusing.